fieldservice/models/fsm_equipment.py

```diff
@@ -102,3 +102,9 @@ def _onchange_stage_id(self):
             self.hide = True
         else:
             self.hide = False
+
+    def copy(self, default=None):
+        if default is None:
+            default = {}
+        default["name"] = self.name + _(" (copy)")
```

Comment:
what does `" (copy)"` string bring?
what is the expected result of this concatenation?

Reply:
ah ok, I see)
`EQ1 (copy)` shouldn't raise an exception